### DISCUSSION TITLE
Welcome to modern-cpp-tutorial Discussions!

### DISCUSSION THREAD

**Discussion by changkun** [changkun/modern-cpp-tutorial#151] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DGC-2K** [comment]
Hello,

I am a mechanical engineer and I have an interest in
C++. I will soon get more involved in the community.

Cheers

Holen Sie sich Outlook für iOS<https://aka.ms/o0ukef>
________________________________
Von: Changkun Ou <notifications@github.com>
Gesendet: Tuesday, December 8, 2020 8:11:55 PM
An: changkun/modern-cpp-tutorial <modern-cpp-tutorial@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [changkun/modern-cpp-tutorial] Welcome to modern-cpp-tutorial Discussions! (#151)


👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  *   Ask questions you’re wondering about.
  *   Share ideas.
  *   Engage with other community members.
  *   Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.
